pallets/loans/src/tests.rs

```diff
@@ -934,3 +898,14 @@ fn get_price_works() {
     );
     MOCK_PRICE_FEEDER::reset();
 }
+
+// Groups all tests that conflict internal storage due to concurrent reads and writes
+// or in other words, groups all flaky concurrent tests.
+#[test]
+fn sequential_tests() {
+    total_collateral_asset_value_works();
+
+    liquidate_borrow::collateral_value_must_be_greater_than_liquidation_value();
+    liquidate_borrow::full_workflow_works_as_expected();
+    liquidate_borrow::liquidator_can_not_repay_more_than_the_close_factor_pct_multiplier();
+}
```

Comment:
`cargo test` spawns `N`, where `N = number of host cores`, jobs so parallel by default and by design.
To avoid having to create a sequential entry-point for tests, it is also possible to always run `cargo test -- --test-threads 1`

Reply:
That makes sense. Maybe this will fix the intermittent problems in Test Coverage. @GopherJ